apps/nextjs-app-router/src/app/[lang]/[[...path]]/layout.tsx

```diff
@@ -13,10 +14,15 @@ export default function RootLayout({
   return (
     <html lang={params.lang}>
       <head>
-        <DraftModeScript />
+        <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
       </head>
       <body>
-        <MakeswiftProvider>{children}</MakeswiftProvider>
+        <MakeswiftProvider
+          previewMode={draftMode().isEnabled}
+          locale={params.lang}
```

Comment:
Have to pass `previewMode` at this level because `draftMode()` can't be used in a client component (and `MakeswiftProvider` has to be a client component so we can pass `runtime` to it)